1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,151 @@
+    I want to edit existing and create new projects
+
+    @2903a
+    Scenario Outline: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        # Save button is removed as Project API demands a PM
+        And they see a Next Button
+
+    @2905
+    Scenario: Curator opens a Project in the Project Wizard
+        # Same end result as "User Edits a Project in the Project Wizard" 
+        # but enables a Curator to edit projects where he's not an participant
+        Given User opens Landing Page for Project
+        And User is Curator on Project's Project Owner's Institution
+        And the project lacks an Approval of type "REK"
+        # A "REK" Approved project is a Health Project. 
+        # See health_related_projects.feature for details.
+        When they click the Edit button
+        Then User sees the Project Wizard
+        And it contains data about the Project
+
+    @2906a
+    Scenario Outline: User opens Participants tab for Project
+        Given User sees the Project Wizard
+        And User has one of these role in the project:
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+        When they click the Participants tab
+        Then they see lists of:
+            | Project Managers     |
+            | Project Participants |
+        And they see Button Add Project Participant
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Button "<Button>" decided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Status    | Button                   |
+            | Draft     | Save draft               |
+            | Published | Update published project |
+
+    @2906b
+    Scenario Outline: User opens Dialog for adding Project Participant
+        Given User opens Participants tab for Project
+        And User has one of these role in the project:
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+        When they click the Add Project Participant Button
+        Then they can select role to be either of:
+            | Project Partcipant    |
+            | Local Project Manager |
+        And they can see User search field
```

Comment:
```suggestion
        And they can see User search field
        Examples:
            | Curator               |
            | Project Owner         |
            | Project Manager       |
            | Local Project Manager |
```

Reply:
Hvis det skal testes på forskjellige roller så må rollen inn i `Examples`. Da vil det bli kjørt en test for hver rolle. Hvis rollene ligger i en datatabell i scenarioen vil det kjøres en test antar at en bruker har alle rollene.